receiver/k8sclusterreceiver/internal/collection/collector.go

```diff
@@ -58,17 +59,17 @@ import (
 // an interface to interact with refactored code from SignalFx Agent which is
 // confined to the collection package.
 type DataCollector struct {
-	logger                   *zap.Logger
```

Comment:
Can you share a little context around why we're moving from using the zap logger here to the `CreateSettings` object? I'm wondering how this change is connected with using the pdata format for metrics instead of opencensus.